libcodechecker/server/database/config_db_model.py

```diff
@@ -113,17 +113,17 @@ def __init__(self, permission, product_id, name, is_group=False):
 
 
 class Session(Base):
-    __tablename__ = 'sessions'
+    __tablename__ = 'user_sessions'
 
-    # Auth-String is a SHA-256 hash, while token is a Python UUID which is
-    # 32 characters (both in hex expanded format).
-    auth_string = Column(CHAR(64), nullable=False, primary_key=True)
-    token = Column(CHAR(32), nullable=False)
+    user_name = Column(String, primary_key=True)
+    token = Column(CHAR(32), nullable=False, index=True)
+    groups = Column(String)
```

Comment:
It would be nice to add some comment that this is a comma separated list or how are the groups stored? What if we have utf8 charaters in the group names?